### PR TITLE
Updated CHANGELOG and package.json for 3.14.2 release

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.2](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.2)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.2](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.2)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.2](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.2)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.2](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.2)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.2](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.14.2)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.14.2 (2020-04-27)
+
+- Fixed modern theme header styles by overriding the default theme's `font-family` and `color` CSS properties. [#122](https://github.com/blackbaud/skyux-theme/pull/122)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:3]: Add "the" between "Fixed" and "modern," and this is good to go.

Comment (Blackbaud-DeniseCisneros) [reply CHANGELOG.md:3]: Chatted about this offline. Disregard previous comment.

**Pull request merged**